apps/cancel-request/translations/src/en/fields.json

```diff
@@ -13,5 +13,25 @@
         "options": {
             "none_selected": "Select a country"
     }
+        },
+    "cnc-who-is-completing": {
+        "legend": "Who completed the original form?",
+        "hint": "This form must be completed by the same person who made the original request or their authorised legal repesentative",
```

Comment:
I assume this should be-  `repesentative`

However, its as per the figma so we can probably raise this with UCD so it can be corrected.

![Screenshot 2024-10-07 at 11 00 36](https://github.com/user-attachments/assets/e2ca0e52-a041-4cb2-93ed-6fd074ba2447)

Reply:
Good spot